### ISSUE TITLE
Vehicle Image Classification

### ISSUE THREAD

**Issue by Soumayan-pal01** [Ayush7614/Soomvaar#146]
## TITLE  - Vehicle Image Classification

Assignee - 'Soumayan-pal01'

### AIM of the project

The aim of this project is to classify a vehicle image which type of vehicle it is. First we will try to visualize the training images to understand how to perform this multiclass image classification. Then we will use Neural Network approaches: MLP and CNN, to train the model and finally compare the validation accuracy of both the models. The dataset contain 7 classes of vehicles.
